### PR TITLE
update npm cypress xpath

### PR DESCRIPTION
**Pull request by mdavis-enquizit**
## Description

Update NPM Cypress Xpath version. 
## Checklist before requesting a review
- [ ] PR focuses on a single story
- [ ] Code has been fully tested to meet acceptance criteria
- [ ] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [ ] All new functions/classes/components reasonably small
- [ ] Functions/classes/components focused on one responsibility
- [ ] Code easy to understand and modify (clarity over concise/clever)
- [ ] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [ ] PR does not contain hardcoded values (Uses constants)
- [ ] All code is covered by unit or feature tests

### CODE REVIEW

File: testing/regression/package-lock.json
Comment (lramakr) [inline testing/regression/package-lock.json:1]: I hope you don't need to check-in this package-lock

Comment (mdavis-enquizit) [reply testing/regression/package-lock.json:1]: https://github.com/CDCgov/NEDSS-Modernization/pull/1531/files

Comment (mdavis-enquizit) [reply testing/regression/package-lock.json:1]: I see the recent bump to file will create conflicts.

**Comment by mdavis-enquizit** [comment]
Closed for https://github.com/CDCgov/NEDSS-Modernization/pull/1531

**Pull request closed without merging**